### ISSUE TITLE
[CAIA Intake] Debt Resolution: Digital Dispute Form

### ISSUE THREAD

**Issue by mgleo** [department-of-veterans-affairs/va.gov-team#96804]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) shared services team. Feel free to delete instructions and unselected options from the ticket to make it easier to scan.

## Your team

- **Product name:** Digital Dispute Form
- **Team name**: Debt Resolution
- **OCTO product owner:** Denise Coveyduc
- **Product manager:** Tom Davis
- **Designer(s):** Megan Gayle, Joseph Lee
- **Slack channel:** #debt-resolution
- **Dedicated content writer on your team (if you have one):** 
- **Dedicated a11y specialist on your team (if you have one):**

## CAIA specialists involved
_Filled out by CAIA_
- **IA:** @kristinoletmuskat 
- **Content:** 
- **Accessibility:** 
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [x] Digitizing a new form
- [x] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [x] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  

### What's the nature of your initiative and desired outcomes?
`If a Veteran has a VA debt that they would like to dispute there is currently no formalized process for them to follow. Disputes can currently be submitted in a variety of formats (verbal, written, etc.) and the information required is not clearly defined. This experience can be confusing for Veterans and inefficient as additional information may need to be collected from the Veteran for the VA to process the dispute effectively. We will be launching the form for VBA only for this initial launch but it will eventually include both VBA and VHA debt`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content. FOR FIGMA/MURAL: please tell us which screens/board/section to look at so we don't leave comments on old stuff. 
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/digital-dispute/product-outline.md)
- [Epic with timeline](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/82003)
- [User flow](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=0-1&t=3A4Fra7C0SGhRzKs-1)
- [Mockups (VBA- initial launch)](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=152-4620&t=3A4Fra7C0SGhRzKs-1)
- [Mockups (VBA and VHA- will be launched in the future)](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=152-4621&t=3A4Fra7C0SGhRzKs-1)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure
- Planning on launching to 5% on Dec 9, 2024 (VBA only)

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Userflow and Mock ups are complete. Prototype is being worked on. Planning on launching to 5% on Dec 9, 2024 (VBA only)`

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [x]  Yes
- [ ]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** 
- (VBA only):

Launch 5% | Dec 9, 2024
Launch 100% | Dec 12, 2024

- **What is the name of your engineer who will build the React widget?** 
- `Andrew Rodiek`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide a link to your Collaboration Cycle Request:
- **[Collaboration Cycle for Debt Resolution - Digital Debt Dispute #95575](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95575)** 

## Next steps
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [x] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document). [Mock ups](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=19-220&t=3A4Fra7C0SGhRzKs-1)
- [ ] Please don't close this intake when work is complete - CAIA will close and track

**Comment by mgleo** [comment]
cc @denisecoveyduc @Tom-Davis @dhill802

**Comment by mgleo** [comment]
Hi @kristinoletmuskat, here is the CAIA intake ticket that @josephrlee spoke to you about. We are looking for assistance on URL names.

**Comment by NaomiPMC** [comment]
@NaomiPMC  confirming from @kristinoletmuskat  if this can be moved to ready or in progress

**Comment by mgleo** [comment]
Must notes from Erin [#96846](https://github.com/department-of-veterans-affairs/va.gov-team/issues/96846)

> Must: define entry points What are the primary path(s) Veterans will take to access this tool?

> Must: show how Veteran can check status of dispute after submission Where can Veterans go to understand the status of their request, post-submission of form? My VA, debt tool, both?

cc @denisecoveyduc @Tom-Davis @dhill802

**Comment by kristinoletmuskat** [comment]
Hey @mgleo and @josephrlee , here's a first pass at the IA stuff we'll need to figure out for this launch in [Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1731461600152/93a4a19b003f01b86534471cb686f81bf073e4a2). 

Would y'all be able to take a look and leave comments or questions as needed?

I remember that there was past work to update the FSR wizard, and I'm not sure where that is in development, and how we should consider those changes when planning these navigation pathways. I think the URLs are pretty straightforward for this form, it's the navigation that is a little trickier. 

My biggest user flow flag is helping the user understand the status of their dispute, and how to submit additional information as needed. I see you want to add document upload in the future, but how are users going to handle this today? And is there a way to indicate on the debt detail page itself that the debt is being disputed?

Since this is a new form, I think it might need a form number? I can try to find out.

FYI @aprocik1 we should connect about who the content person will be on this, given the quick turnaround.

**Comment by laurwill** [comment]
Hi @kristinoletmuskat and @mgleo ,

I think I can clear up some of the confusion here, but also thinking we should sync on this — this is a very tight turnaround you all are aiming for, and there's quite a bit of IA and content thinking to here.

The new [Request help with VA debt from benefit overpayments page](http://preview-prod.vfs.va.gov/preview?nodeId=67350) is currently in SME review, and we were aiming to launch it as soon as possible, but timing depends on SME approval. Once that page and related page updates are live, we'll remove the wizard from the FSR form entirely. So we shouldn't use that wizard as an entry point or make any updates to that wizard content. We'll need to assess whether we can get the new static page launched in time for this form release, or whether we need to place the form on the debt detail page only to start, and then add to static page later.

Let's sync to talk about what's possible in terms of timing and placement — @kristinoletmuskat I have a sync about the static page work with Megan and Charlotte at 2:30 tomorrow. I added you but it looks like you're busy at that time. If you can't make it, we can find a time to meet on Friday.

@mgleo @josephrlee a few questions and notes about the Figma designs in the meantime:
- The intro page is using an outdated template — we'll need to update to use the (much simpler!) non-application version of the [intro page template](https://design.va.gov/templates/forms/introduction). This removes the process list and provides a few brief bullets of need-to-know info before people start the form.
- Does this form require verified accounts to sign in? If so, we'll need to update the sign-in alert to the new [component version here](https://design.va.gov/components/alert/alert-sign-in/), and work with the identity team to make sure the sign-in and verification flows are working correctly for all account types.
- The "VBA only" flow in Figma includes a [copay bill](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=43-4650&t=9MG9lIvf5vYeGxuE-1). Can you confirm if you're including copay bills in the initial launch, or only including benefit overpayments?
- For the collab cycle item "Must: show how Veteran can check status of dispute after submission Where can Veterans go to understand the status of their request, post-submission of form? My VA, debt tool, both?", we need to clarify if we're talking about **status of submitted dispute form** or **status of the actual dispute.** These are very different things, and showing status of the actual dispute (from receipt to decision) would be a much bigger project. Are we only talking about confirming that we've received the submitted online dispute form, and that there was no silent failure of the form submission?

**Comment by mgleo** [comment]
Hi @kristinoletmuskat, Thank you for creating the Mural, this looks great. I'll add comments/questions today. 
- Here is the link to the Wizard / static page ticket: [#92685](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92685)
- We don't have plans to implement a document upload feature currently. Currently, we would like users to enter information regarding why they are disputing debt in a text field. 
- We do have a VBA status message on the [debt portal](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=0-10322&t=qnRizVW7wdgTMAJm-1) that appears on a specific debt if they have disputed it. [On Code 822](https://docs.google.com/spreadsheets/d/1h8CYQH9W7ka2llPsHX50MQ7Qy5uoKFsLmMtNlQ6eLfk/edit?usp=sharing), it reads "Continue making monthly payments while we review your dispute
To avoid late fees or further collection action on your bill..."

Hi @laurwill 
- Thanks for the updated intro page template!
- The VBA flow should only show VBA debt. I saw a copay was listed on the Review page, just corrected that. Thank you for asking. 

I'm not sure of the answers to the questions below. I believe we should require verified accounts to sign in. CCing @denisecoveyduc @Tom-Davis @dhill802 for confirmation.

> Does this form require verified accounts to sign in? If so, we'll need to update the sign-in alert to the new [component version here](https://design.va.gov/components/alert/alert-sign-in/), and work with the identity team to make sure the sign-in and verification flows are working correctly for all account types.

> For the collab cycle item "Must: show how Veteran can check status of dispute after submission Where can Veterans go to understand the status of their request, post-submission of form? My VA, debt tool, both?", we need to clarify if we're talking about status of submitted dispute form or status of the actual dispute. These are very different things, and showing status of the actual dispute (from receipt to decision) would be a much bigger project. Are we only talking about confirming that we've received the submitted online dispute form, and that there was no silent failure of the form submission?
@erinrwhite can you confirm this information ^?

**Comment by kristinoletmuskat** [comment]
@mgleo thank you! Just to confirm on the 822 code -- the card shows up in the debt list page (left image), and the alert shows up on the detail page (right image), is that correct? Just want to double check I understand where they're used!

![Image](https://github.com/user-attachments/assets/2c17fe67-fadf-4b29-8344-3956b907d2fc)

**Comment by erinrwhite** [comment]
@mgleo 

> Are we only talking about confirming that we've received the submitted online dispute form, and that there was no silent failure of the form submission?

Yes, thanks for clarifying this - just want to make sure that if there is a failure after this particular form is submitted, users are notified.

**Comment by josephrlee** [comment]
@kristinoletmuskat, yes, that's correct, the `va-alert` shows up on the detail page and the `va-card` shows up on the summary page. 

cc: @mgleo

**Comment by kristinoletmuskat** [comment]
Hey @mgleo and @josephrlee , here is the final [IA spec](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/dispute-debt.md) for this dispute form. 

Could you take a look and see if anything looks wrong to you?

cc @laurwill

**Comment by josephrlee** [comment]
@kristinoletmuskat, I just put the sub-urls into a list to make it more readable, but everything else looks great! Thanks!

**Comment by artsymartha68** [comment]
"Normal" is the correct priority, untagging myself.

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mnorthuis** [comment]
@VCM86 This would be for a DE writer and IA.  Can you review status and determine next steps on this work?

**Comment by VCM86** [comment]
@rorycooper -- assigning this to you. Please review the ticket, and we'll schedule a meeting with Danielle to discuss next steps when I'm back. Thank you!

**Comment by mnorthuis** [comment]
@VCM86 @rorycooper Please include IA - @jonichan-ia - in conversations for awareness.

**Comment by HeatherRienks** [comment]
following

**Comment by mgleo** [comment]
Hi @VCM86, @jonichan-ia, @rorycooper,
Update on this work: Laura did look at and make a lot of suggestions on our [Dispute Form Figma file](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form--VHA-VBA-?node-id=152-4836&t=nbsp4HEh70rtsxQ6-1) last year so we should be ok for now!

We will be starting a research study this week that combines both the Digital Dispute form and the new Wizard/R&S page. 
[Here is a link to the prototype](https://zyc78g.csb.app/manage-va-debt). 

**Wizard/R&S page:** We will need assistance in adding links/info for the new R&S page on the unauth [Manage VA Debt page](https://va.gov/manage-va-debt/) since it is an auth/Drupal page.

**Comment by mistymg** [comment]
@mgleo @VCM86 @rorycooper I was looking through the Content team's board and found [this old ticket](https://github.com/orgs/department-of-veterans-affairs/projects/929/views/78?pane=issue&itemId=86125015&issue=department-of-veterans-affairs%7Cva.gov-team%7C96539) from a former team member that might be relevant to this work.

**Comment by rorycooper** [comment]
Confirmed that this ticket can be closed. [Slack convo](https://dsva.slack.com/archives/CPE4AJ6Q0/p1747244432668189)
